cmd/cli/operations/update_job_test.go

```diff
@@ -11,39 +11,41 @@ import (
 )
 
 /*
- * filterRunningJobs
+ * filterRunningJobsByName
  */
 func TestFilterRunningJobsShouldReturnAnEmptySliceIfNoJobsWereProvided(t *testing.T) {
 	operator := RealOperator{}
 
-	res := operator.filterRunningJobs([]flink.Job{})
+	res := operator.filterRunningJobsByName([]flink.Job{}, "test")
 
 	assert.Len(t, res, 0)
 }
 
 func TestFilterRunningJobsShouldReturnAnEmptySliceIfNoJobsAreRunning(t *testing.T) {
 	operator := RealOperator{}
 
-	res := operator.filterRunningJobs([]flink.Job{
+	res := operator.filterRunningJobsByName([]flink.Job{
```

Comment:
I'm not too happy about the formatting. Could you put the `[]flink.Job{` on the next line and inline it accordingly. Something like this:

```
	res := operator.filterRunningJobs(
		[]flink.Job{
			flink.Job{
				Status: "STOPPED",
			},
		},
		"test",
	)
```

Could you do the same with the other tests please?

Reply:
Done!